### PR TITLE
Disolving FavTriggersFragment into TriggersFragment #147

### PR DESCRIPTION
**Pull request by pg301**

### CODE REVIEW

**Comment by pilhuhn** [comment]
@pg301 While the fix for #145 is a tiny change and it feels cumbersome to send a PR just to add a typo, it is a good practice to do so nevertheless. The situation is that later someone looks at the commits/changes in master and sees the change in `AndroidManifest.xml` and wonders what this has to do with the Fav triggers and why this is needed or not. 
Sending one PR per issue makes this more clear (unless more than 1 issue have the same cause/fix).
This also applies to e.g. code format changes, bumping the year in the header or similar.

**Comment by KeenWarrior** [comment]
@pilhuhn It is mainly her rebasing issue. That was her work from last PR

**Comment by KeenWarrior** [comment]
@pg301 When Your PR is merged you need 
1. switch back to your master branch
2. pull from upstream into master
3. create new branch to start new work

**Comment by KeenWarrior** [comment]
@pg301 Always take care to create new branch for PR from your master branch

File: mobile/src/main/java/org/hawkular/client/android/activity/MainActivity.java [hunk 1]
```diff
@@ -303,8 +302,14 @@ private void showFavourites() {
...
         adapter.reset();
+
+        Bundle bundle = new Bundle();
+        bundle.putString("state", "From Favourite");
+        TriggersFragment triggersFragment = new TriggersFragment();
+        triggersFragment.setArguments(bundle);
+
         adapter.addFragment(new FavMetricsFragment(), "Metrics");
-        adapter.addFragment(new FavTriggersFragment(), "Triggers");
+        adapter.addFragment(triggersFragment, "FavTriggers");
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/activity/MainActivity.java:312]: Leave the name to `Triggers`

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java [hunk 2]
```diff
@@ -114,14 +116,19 @@ public void onRefresh() {
     }
 
     private void setUpTriggersForced() {
+
+        if( getArguments().getString("state").compareTo("From Favourite") == 0)
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java:120]: there should always be brackets around if and else. 

For consistency use opening bracket at same line of starting of block 

eg : `private void setUpFavTriggers() {`

**Comment by pg301** [comment]
@pilhuhn I got your point and will keep this in mind. @garg-anuj Thanks for the review. I have made these changes and will open a new PR addressing just this issue.

**Comment by KeenWarrior** [comment]
In regard of #150

**Pull request closed without merging**